app/Services/CustomerIo.php

```diff
@@ -24,6 +24,16 @@ public function __construct()
         ]);
     }
 
+    /**
+     * Is Customer.io enabled for this app?
+     *
+     * @return bool
+     */
+    protected function enabled(): bool
+    {
+        return config('features.blink');
```

Comment:
I'm planning to switch the name of this config & environment variable in both applications later (but don't want to introduce a change just yet that'd require changing any settings in order to deploy this).